.github/workflows/ci.yml

```
@@ -68,8 +68,7 @@ jobs:
     strategy:
       matrix:
         golang:
-          - 1.14.x
-          - 1.15.x
+          - 1.18.x
```

```suggestion
          - 1.18.x
          - 1.19.x
```

On most projects, it's interesting to ensure that a go project works well on the two last stable releases.